improved exception handling for no handler (controller) found for the

SrdjanStevanetic opened the pull request:
path

SrdjanStevanetic committed:
improved exception handling for no handler (controller) found for the
path

gsergiu commented on entity-management-web/src/main/resources/entitymanagement.properties:
```diff
@@ -89,4 +89,7 @@ batch.step.throttleLimit = 10
+zoho.sync.batch.size = 200
+
+
+spring.resources.static-locations=classpath:/public/
```
not needed in properties, it can be hardcoded

gsergiu commented on entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMExceptionHandler.java:
```diff
@@ -97,4 +100,21 @@ public ResponseEntity<EuropeanaApiErrorResponse> handleInvalidMediaType(
         .body(response);
   }
+
+  @ExceptionHandler(NoHandlerFoundException.class)
```
integration test needs to be added

gsergiu reviewed (commented):
an integration test for the changes needs to be added

SrdjanStevanetic committed:
1) test for exception handling of no existing url
(NoHandlerFoundException); 2)Merge branch 'develop' into
EA-3377-handle-not-found-exception

Conflicts:
	entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMExceptionHandler.java

SrdjanStevanetic committed:
comments on the exception handling test
